app/src/main/java/com/github/onlynotesswent/ui/overview/Overview.kt

```diff
@@ -108,10 +113,31 @@ fun OverviewScreen(
             userViewModel = userViewModel,
             context = context,
             navigationActions = navigationActions)
+        if (showCreateNoteDialog) {
+          NoteDialog(
```

Comment:
This code is the same as the one in Overview , but I think it's fine to leave it as is. Extracting the common code would make too much modularization, which might only make the code messier.